crates/ruff_dev/src/generate_options.rs

```diff
@@ -3,6 +3,7 @@
 //! Used for <https://docs.astral.sh/ruff/settings/>.
 use std::fmt::Write;
 
+use ruff_python_trivia::textwrap;
```

Comment:
Ohh I didn't realize when you asked this that it was specifically for Ruff :joy:

Reply:
:D